### PR TITLE
[zh] Translate create hostprocess pod

### PR DESCRIPTION
**Pull request by tengqm**
/language zh

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+-->
+类似于安装安全补丁、事件日志收集等这类管理性质的任务可以在不需要集群操作员登录到每个
+Windows 节点的前提下执行。HostProcess 容器可以以主机上存在的任何用户账户来运行，
+也可以以主机所在域中的用户账户运行，这样管理员可以通过用户许可权限来限制资源访问。
+尽管文件系统和进程隔离都不支持，在启动容器时会在主机上创建一个新的卷，
+为其提供一个干净的、整合的工作空间。HostProcess 容器也可以基于现有的 Windows
+基础镜像来制作，并且不再有 Windows 服务器容器所带有的那些
+[兼容性需求](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility)，
+这意味着基础镜像的版本不必与主机操作系统的版本匹配。
+不过，仍然建议你像使用 WIndows 服务器容器负载那样，使用相同的基础镜像版本，
```
Comment (sftim) [inline content/zh/docs/tasks/configure-pod-container/create-hostprocess-pod.md:59]: nit: 
```suggestion
不过，仍然建议你像使用 Windows 服务器容器负载那样，使用相同的基础镜像版本，
```

File: content/zh/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+Resource limits (disk, memory, cpu count) are applied to the job and are job wide.
+For example, with a limit of 10MB set, the memory allocated for any HostProcess job object
+will be capped at 10MB. This is the same behavior as other Windows container types.
+These limits would be specified the same way they are currently for whatever orchestrator
+or runtime is being used. The only difference is in the disk resource usage calculation
+used for resource tracking due to the difference in how HostProcess containers are bootstrapped.
+-->
+## 资源约束    {#resource-limits}
+
+资源约束（磁盘、内存、CPU 个数）作用到 Job 之上，并在整个 Job 上起作用。
```
Comment (sftim) [inline content/zh/docs/tasks/configure-pod-container/create-hostprocess-pod.md:315]: I would avoid writing Job in title case; a reader might confuse this with the Kubernetes API kind named Job.
Another option might be to add some extra wording to clarify.

**Review by sftim — COMMENTED** [review]
Two minor bits of feedback.

File: content/zh/docs/tasks/configure-pod-container/create-hostprocess-pod.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/create-hostprocess-pod.md:315]: So ... the job in this context is actually a workload, not necessarily a Job resource? I may have mistaken the "job" word here.

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**